content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -14,11 +14,18 @@ is generally used by cluster components to validate the API server's
 certificate, by the API server to validate kubelet client certificates, etc. To
 support this, the CA certificate bundle is distributed to every node in the
 cluster and is distributed as a secret attached to default service accounts.
+
 Optionally, your workloads can use this CA to establish trust. Your application
 can request a certificate signing using the `certificates.k8s.io` API using a
 protocol that is similar to the
 [ACME draft](https://github.com/ietf-wg-acme/acme/).
 
+Note: Certificates signed using the `certificates.k8s.io` API are not
+guaranteed to be signed by the cluster root CA. The CA used for signing depends
+on the cluster setup. CA certificate bundle distributed to nodes and service
```

Comment:
If possible, reword this to use the active voice and to [address the reader as “you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you).